firebase.json

```diff
@@ -403,6 +404,7 @@
 
       { "source": "/tools/**/download{,.html,/**}", "destination": "/get-dart", "type": 301 },
       { "source": "/tools/analyzer", "destination": "/tools/dart-analyze", "type": 301 },
+      { "source": "/tools/dart-mcp-server", "destination": "/tools/mcp-server", "type": 301 },
```

Comment:
@parlough this is submitted and tested this. The `/to/mcp-server` redirect works, but I'm not having luck with this one. Not sure if I'm using this link incorrectly or if there is some other issue.